src/librustdoc/html/static/main.js

```diff
@@ -161,17 +161,18 @@ function getSearchElement() {
         return window.history && typeof window.history.pushState === "function";
     }
 
+    function isHidden(elem) {
+        return elem.offsetHeight === 0;
+    }
+
```

Comment:
This function seems odd to have, there's code further down which called an `isHidden()` function you don't appear to have removed, so is this shadowing one?

Reply:
You're absolutely right! I removed the old one.